libs/components/forms/src/lib/modules/input-box/input-box.component.spec.ts

```diff
@@ -678,11 +686,47 @@ describe('Input box component', () => {
 
       expect(characterCountLabelEl).toHaveText('3/10');
 
+      // Aria-label updates when not focused
+      expect(els.labelEl?.getAttribute('aria-label')).toBe(
```

Comment:
This is verbose enough and repeated enough times to warrant its own function.

Reply:
Pulled this out to a helper function.